SECP-service/src/main/resources/assets/app/css/app.css

```diff
@@ -1,5 +1,46 @@
+@import url("https://fonts.googleapis.com/css?family=Raleway:400,700");
 body {
   margin-top: 60px;
   padding: 10px;
   background-color: #ECF0F1;
+  font-size: 16px;
+  color: #333333;
+  background-color: #ffffff;
 }
+ .navbar-inverse {
```

Comment:
Do you mind reverting back to the default navbar color for the website. Everything is just white right now. It does not look good. :)